modules/store-devtools/src/extension.ts

```diff
@@ -80,7 +80,8 @@ export class DevtoolsExtension {
   constructor(
     @Inject(REDUX_DEVTOOLS_EXTENSION) devtoolsExtension: ReduxDevtoolsExtension,
     @Inject(STORE_DEVTOOLS_CONFIG) private config: StoreDevtoolsConfig,
-    private dispatcher: DevtoolsDispatcher
+    private dispatcher: DevtoolsDispatcher,
+    private ngZone: NgZone
```

Comment:
```suggestion
    private dispatcher: DevtoolsDispatcher
```

Add private class member:

```ts
private zoneConfig: ZoneConfig;
```

in the constructor:

```ts
this.zoneConfig = injectZoneConfig(config.connectInZone);
```